.github/workflows/pre-release-pr.yml

```diff
@@ -0,0 +1,83 @@
+name: Release test pre-release version
+
+on:
+  repository_dispatch:
+    types: [pre-release-command]
```

Comment:
Could we not use a `workflow_dispatch` event w/o the bot?

Reply:
I just used what the library suggested, do you know why we would prefer workflow_dispatch over repository_dispatch? It has been a while since I used the former so I am not 100% sure why it exists

Reply:
If that's what the library suggests, then we may as well just stick to it. My understanding is that `workflow_dispatch` is for github->github events, while `repository_dispatch` is for external->github events.

Reply:
I will read about this a bit and see what makes more sense